### PR TITLE
Create game.tscn

### PR DESCRIPTION
**Pull request by asafasaf**
Added Base Node2D

### CODE REVIEW

**Review by nxidplease — APPROVED** [review]
You are a noob

**Comment by tomsandalon** [comment]
#forcemerge

**Comment by tomsandalon** [comment]
#skipreview #automerge

**Comment by tomsandalon** [comment]
#noobmode

File: Scenes/game.tscn [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[gd_scene format=3 uid="uid://ci885nw14tvg5"]
+
+[node name="Game" type="Node2D"]
```
Comment (tomsandalon) [inline Scenes/game.tscn:3]:
```suggestion
[gd_scene format=3 uid="uid://ci885nw14tvg5"]
[Gayzer]
[node name="Game" type="Node2D"]
```

**Review by tomsandalon — CHANGES_REQUESTED** [review]

**Pull request merged**